registration-processor-default.properties

```diff
@@ -999,3 +999,5 @@ mosip.registration.processor.print.template=RPR_UIN_CARD_TEMPLATE
 
 # this property is used to define issuers eg partnerid:credentialType:templateTypeCode
 mosip.registration.processor.issuer=mpartner-default-print:${mosip.registration.processor.print.credentialtype}:${mosip.registration.processor.print.template};mpartner-default-digitalcard:${mosip.registration.processor.digitalcard.credentialtype}:${mosip.registration.processor.digitalcard.template}
+##timeout in milliseconds for health check registrer
+mosip.regproc.health-check.handler-timeout=2000
```

Comment:
why 5 seconds as default and 2 seconds in the config ?

Reply:
Can we keep the proper default in the code and avoid adding it in the config ?

Reply:
Fixed